test/config-authentication-oidc/features.yaml

```diff
@@ -17,7 +17,9 @@ kind: ConfigMap
 metadata:
   name: config-features
   namespace: knative-eventing
+  labels:
+    knative.dev/config-propagation: original
+    knative.dev/config-category: eventing
```

Comment:
I'm curious, what are these labels needed for?

Reply:
Same as https://github.com/knative/eventing/pull/7622#discussion_r1465085559. But let me remove it...

Reply:
Updated